bundles/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/communication/RokuCommunicator.java

```diff
@@ -114,6 +118,9 @@ public DeviceInfo getDeviceInfo() throws RokuHttpException {
             JAXBContext ctx = JAXBUtils.JAXBCONTEXT_DEVICE_INFO;
             if (ctx != null) {
                 final String response = getCommand(urlQryDevice);
+                if (logger.isTraceEnabled()) {
+                    logger.trace("Called {}, got response: {}", urlQryDevice, response);
+                }
```

Comment:
What is the reason for manually checking the log level? Since no expensive operations are involved, but only simple logging of local variables, this should be sufficient:
```suggestion
                logger.trace("Called {}, got response: {}", urlQryDevice, response);
```

Reply:
> What is the reason for manually checking the log level? Since no expensive operations are involved, but only simple logging of local variables..

That was the idea... to avoid expensive operations. But since none are involved, I have removed the checks.